h2o-core/src/main/java/hex/Model.java

```diff
@@ -166,7 +166,14 @@ else if (_output.nclasses() > 1)
   }
 
   public ToEigenVec getToEigenVec() { return null; }
-
+  
+  /**
+   * Intended for overriding in SharedTreeModel for tree visualization.
+   */
+  public void visualize(String output) {
```

Comment:
I think it would be better to have an interface for this method, or we can add it to sharedtreemodel or simmilar place and check with instanceof and return an error when user tries to visualise an usupported model

Reply:
I need to use this from core and in core sharedtreemodel is not visible so I cant use instanceof. I think I can do interface 👍

Reply:
why do you need to call it on model? you can make ModelPngWriter make to accept not Model but lets say ModelWithVisualisation

Reply:
i see, did that interface